src/include/homestore/replication/repl_dev.h

```diff
@@ -49,6 +49,12 @@ VENUM(journal_type_t, uint16_t,
       HS_CTRL_REPLACE = 3, // Control message to replace a member
 )
 
+// Whether block allocation type is a single multiblkid or vector of blkids.
```

Comment:
Nit: we probably can move this into `repl_req_cxt` to make it only visiable to itself, not to callers.